swapi_api/route/index.ts

```diff
@@ -1,21 +1,16 @@
-import { createWebHistory, createRouter, RouteRecordRaw } from "vue-router";
+import { createWebHistory, createRouter } from "vue-router";
+import Home from '../layouts/Home.vue'; 
+const routes =[
 
-import main from '../layouts/main.vue'; 
-
-const routes: Array<RouteRecordRaw> = [
     {
-        path: '/',
-        name: 'home',
-        component: () => import('../views/Home.vue'),
-        meta:{ 
-            layout: main
-        }
+        path:'/',
```

Comment:
! formatowanie

Maciek, rozmawialiśmy o tym, pokazywałem nawet jako przykład to...